src/stream/src/executor/dispatch.rs

```diff
@@ -162,6 +164,12 @@ impl DispatchExecutorInner {
     async fn dispatch(&mut self, msg: Message) -> Result<()> {
         match msg {
             Message::Chunk(chunk) => {
+                let actor_id_str = self.actor_id.to_string();
```

Comment:
pre-calculate means that we should put this in struct body, so that throughout the actor lifetime this string will only be generated once.